scrinium.csl

```diff
@@ -0,0 +1,794 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Scrinium: Journal of Patrology and Critical Hagiography</title>
+    <title-short>SCRI</title-short>
+    <id>http://www.zotero.org/styles/scrinium</id>
+    <link href="http://www.zotero.org/styles/scrinium" rel="self"/>
+    <issn>1817-7530</issn>
+    <eissn>1817-7565</eissn>
+    <link href="brill.com/scri" rel="documentation"/>
```

Comment:
Definitely use a full link, i.e. `http://brill.com/scri`. More generally, I don't see a reason not to link to the pdf directly i.e. `http://www.brill.com/sites/default/files/ftp/authors_instructions/SCRI.pdf`